### ISSUE TITLE
Error using grab button with mouse use too

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#143]
I am getting an assertion when I drag the book a little, and then immediately try to tab to the grab button. I get the following error.

```
Uncaught Error: Assertion failed: trying to set focus on a node with invisible accessible content
    at window.assertions.assertFunction (assert.js:22)
    at Rectangle.focus (Accessibility.js?bust=1541545150679:618)
    at HTMLBodyElement.globalKeyListener (AccessibleInstance.js?bust=1541545150679:169)
window.assertions.assertFunction @ assert.js:22
focus @ Accessibility.js?bust=1541545150679:618
globalKeyListener @ AccessibleInstance.js?bust=1541545150679:169
```

I think this is because the mouse interaction is causing visibility to update somehow, but I haven't had time to look into it further.

**Comment by zepumph** [comment]
From #142.

**Comment by zepumph** [comment]
I tried adding this listener to the `wrappedNode` in `GrabButtonNode`:

```js
        focus: () => {

          // If this element is invisible, and it is attempted to be foucsed, then just focus the next item in the DOM
          if ( !wrappedNode.accessibleVisible ) {
            AccessibilityUtil.getNextFocusable().focus();
          }
        },
```

But it didn't work because the assertion is being triggered from the "global listener" in `AccessibleInstance`. Maybe we can sort this kind of thing out with https://github.com/phetsims/scenery/issues/888, because we are centralzing all globally added event listeners into the same system as what will call input listeners for individual Node listeners.

**Comment by zepumph** [comment]
Because of the redesign work done in https://github.com/phetsims/scenery-phet/issues/421. This is no longer an issue. I think it is because the `parentNode` that gets past into the a11y interaction node, becomes the grab button instead of the draggable. This way we no longer click and drag something that is accessibleVisible, such that when we press tab, the browser tries to focus it, even though it isn't in the PDOM.
